ui/config/general_config.js

```diff
@@ -103,5 +103,31 @@ UserConfig.registerOptions('general', {
         options['DisplayLanguage'] = value;
       },
     },
+    {
+      id: 'DisableAutomaticOpcodeUpdates',
+      name: {
+        en: 'Disable automatic FATE/CE opcode updates',
+      },
+      type: 'checkbox',
+      default: false,
+    },
+    {
+      id: 'UseCustomOpcodeSource',
+      name: {
+        en: 'Use custom opcode source',
+      },
+      type: 'checkbox',
+      default: false,
+      debugOnly: true,
+    },
+    {
+      id: 'CustomOpcodeSourceUrl',
```

Comment:
Do we need both "use custom opcode source" and "custom opcode source url"? Could these be the same with "blank=don't use"?